fix(launchpad): make sure the correct primary path is used

RolandSherwin opened the pull request:
Fixes https://github.com/maidsafe/safe_network/issues/1888
- When dealing with unix, every mount point would start with '/'. This would lead to incorrect result when selecting the primary partition. So instead, find the mount point that ends with '/'. This would be our primary partition.
- Tested on windows and mac, and it works as intended.

bochaco commented:
bors try

dirvine commented on sn_node/examples/config_handling.rs:
```diff
@@ -25,13 +25,15 @@ async fn main() -> Result<()> {
     // TODO: Uncomment the below lines once we enable reading config from disk
-    // file_config.network_config.local_ip = Some(
-    //     "192.168.0.1"
+    // file_config.local_addr = Some(
```
I know this is commented but seems wrong anyway to have a 192 address there

bochaco replied:
I'll change it to `127.0.0.1`

bochaco commented:
bors r+

RolandSherwin committed:
fix(launchpad): make sure the correct primary path is used
- when dealing with unix, every mount point would start with '/', so
instead find the point that ends with '/'. This would be our primary
partition.
